FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.JobManagement/AzureStorageJobQueueClient.cs

```diff
@@ -301,7 +319,7 @@ public async Task<JobInfo> DequeueAsync(byte queueType, string worker, int heart
             // step 5: skip it if the job is running and still active
             if (jobInfo.Status == JobStatus.Running && jobInfo.HeartbeatDateTime.AddSeconds(heartbeatTimeoutSec) > DateTime.UtcNow)
             {
-                _logger.LogWarning($"Job {jobInfo.Id} is still active.");
+                _logger.LogError($"Job {jobInfo.Id} is still active.");
```

Comment:
Typo in log information for dequeue successfully. Use `Dequeue job {jobInfo.Id} Successfully.` instead.

Reply:
Updated it, thanks.